_sass/_mixins.scss

```diff
@@ -0,0 +1,12 @@
+@mixin transition($transition...) {
```

Comment:
`transition` też nie trzeba prefixować wg http://shouldiprefix.com, jest jakiś inny powód żeby to zostawić?